.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     macos:
-      xcode: '11.4.0'
+      xcode: '12.0.0'
```

Comment:
Had to bump because of:

```
error ava@3.9.0: The engine "node" is incompatible with this module. Expected version ">=10.18.0 <11 || >=12.14.0 <12.17.0 || >=12.17.0 <13 || >=14.0.0". Got "13.13.0"
 
error Found incompatible module.
```

If we can't bump this for some reason I can set it up with GH actions